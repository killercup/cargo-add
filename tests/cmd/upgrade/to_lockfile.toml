bin.name = "cargo-upgrade"
args = ["upgrade", "--workspace", "--to-lockfile"]
status = "success"
stdout = """
one:
three:
two:
four:
"""
stderr = """
    Upgrading libc v0.2.28 -> v0.2.62
    Upgrading rand v0.3 -> v0.3.23
    Upgrading libc v0.2.28 -> v0.2.62
    Upgrading libc v0.2.28 -> v0.2.62
    Upgrading rand v0.2 -> v0.2.1
    Upgrading libc v0.2.28 -> v0.2.62
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
