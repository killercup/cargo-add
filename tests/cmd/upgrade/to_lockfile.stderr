    Upgrading libc v0.2.28 -> v0.2.62
    Upgrading rand v0.3 -> v0.3.23
    Upgrading libc v0.2.28 -> v0.2.62
    Upgrading libc v0.2.28 -> v0.2.62
    Upgrading rand v0.2 -> v0.2.1
    Upgrading libc v0.2.28 -> v0.2.62
